### ISSUE TITLE
"Friends" System

### ISSUE THREAD

**Issue by Fyrsta** [TranxCraft/iTranxCraft#26]
A way for players to add and manage friends in-game.

**Comment by Wild1145** [comment]
Would there be anything that would happen as a result of being a "friend"?

**Comment by Fyrsta** [comment]
@Wild1145 Of course, however I haven't thought of much as of now about it. Most likely there will be a sidebar in-game that will show you the friends you currently have online.

**Comment by Wild1145** [comment]
@WickedGamingUK That would be cool. Depending on how you do some of the other features like the auction system, you may be able to make it display it in bold or for longer or something like that to people listed as "Friends"? Then also have a message display when they login?

**Comment by Fyrsta** [comment]
@Wild1145 That sounds like a good idea.

**Comment by Kent55** [comment]
@WickedGamingUK This would be nice, especially for players to view and chat to friends via the web panel.
